src/Components/Admins/index.js

```diff
@@ -1,24 +1,11 @@
 import { useEffect, useState } from 'react';
 import styles from './admins.module.css';
 import Button from '../Shared/Button';
-import TextInput from '../Shared/TextInput';
 import Modal from '../Shared/Modal';
+import { Link } from 'react-router-dom/cjs/react-router-dom';
```

Comment:
both refer to the same package but the path is different, and I don't know if that can cause future problems
```suggestion
import { Link } from 'react-router-dom';
```